### PR TITLE
[FRR] Force disable next hop group support

### PR DESCRIPTION
**Pull request by nazariig**
Signed-off-by: Nazarii Hnydyn <nazariig@nvidia.com>

Closes https://github.com/sonic-net/sonic-buildimage/issues/17345

This W/A was proposed by Nvidia FRR team before the long term solution is ready.

#### Why I did it
* A W/A to fix default route installation during LAG member flap

##### Work item tracking
* N/A

#### How I did it
* Disabled FRR next hop group support

#### How to verify it

1. Do LAG member flap

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211
- [x] 202305

#### Tested branch (Please provide the tested image version)

- [x] master 
- [x] 202305 

#### Description for the changelog

* N/A

#### Link to config_db schema for YANG module changes

* N/A

#### A picture of a cute animal (not mandatory but encouraged)
```
      .---.        .-----------
     /     \  __  /    ------
    / /     \(  )/    -----
   //////   ' \/ `   ---
  //// / // :    : ---
 // /   /  /`    '--
//          //..\\
       ====UU====UU====
           '//||\\`
             ''``
```

### CODE REVIEW

**Review by dgsudharsan — APPROVED** [review]

**Comment by dgsudharsan** [comment]
@StormLiangMS Can you please review and signoff

**Comment by dgsudharsan** [comment]
@StormLiangMS Can you please review and signoff

**Comment by dgsudharsan** [comment]
Tested on top of 202305 commit https://github.com/sonic-net/sonic-buildimage/tree/c8750621779aaa847c1dc1672a48d467071298a6

**Review by prsunny — APPROVED** [review]

**Comment by prsunny** [comment]
Please wait for @StormLiangMS's approval

**Comment by lizhijianrd** [comment]
Hi @yxieca, can you please help to add "Approved for 202311 Branch" tag to this PR? thanks.

**Pull request merged**
